src/pages/careers.tsx

```diff
@@ -0,0 +1,287 @@
+
+    p {
+        margin-bottom: 40px;
+    }
+
+    h3 {
+        margin: 0;
+    }
+
+    h4 {
+        margin-top: 70px;
+        font-weight: bold;
+    }
+
+    li {
+        position: relative;
+        margin-bottom: 15px;
+        list-style: none;
+
+        &::before {
+            content: '';
+            position: absolute;
+            top: 50%;
+            left: -40px;
+            display: block;
+            background: #1aa6e4;
+            height: 2px;
+            width: 22px;
+
+        }
+
+    }
+
+`
+
+export default class CareersPage extends React.Component<{}, CareersPageState> {
+
+    constructor(props: {}) {
+        super(props);
+        this.state = {
+            isModalOpen: false
+        }
+        this.openModal = this.openModal.bind(this);
+    }
+
+    openModal() {
+        this.setState({ isModalOpen: true })
+    }
+
+    render() {
+        return <IndexLayout title="Gitpod - Careers" canonical="https://www.gitpod.io/careers">
+            <Page>
+                <Container>
+                    <GitGraph left={-900} top={40} graph={[
+                        { start: [0, 115] },
+                        { right: 860 },
+                        {},
+                        { right: 10 },
+                        { down: 200 },
+                        {
+                            fork: [
+                                { right: 115 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                }
+                            ]
+                        },
+                        { down: 205 },
+                        {},
+                        { down: 600 },
+                        {},
+                        { down: 1725 },
+                        { right: 400 },
+                        {}
+                    ]} />
+                    <Logos logos={[
+                        [1050, 125, 40],
+                    ]} />
+                    <div style={{ marginTop: 60 }}>
+                        <h4 style={{ color: colors.fontColor2 }}>We're Hiring</h4>
```

Comment:
Please use a lowercase `h` in `hiring`:

```suggestion
                        <h4 style={{ color: colors.fontColor2 }}>We're hiring</h4>
```